graph/context.js

```diff
@@ -42,6 +42,32 @@ const decorateContextPlugins = (context, contextPlugins) => {
   );
 };
 
+/**
+ * Some pieces of the Context are quite complex to setup, using multiple merges
+ * and other lodash functions. This proxies that access such that it is only
+ * loaded if it is used. Helpful for a query that only uses a loader, and not a
+ * mutator.
+ *
+ * @param {Object} ctx the graph proxy
+ * @param {Function} loader the loadable component that should be proxied
+ */
+const proxyContextLoader = (ctx, loader) =>
```

Comment:
Let's call it `createLazyContextLoader` or similar. It took me a bit to figure out what it does.